src/Pages/NotesPage/NotesPage.jsx

```diff
@@ -1,13 +1,16 @@
 import "./NotesPage.css";
 import {Header,Sidebar,Footer} from "../../components/index";
 import { MainArea } from "./MainArea/MainArea";
+import { ToastContainer} from 'react-toastify';
+import 'react-toastify/dist/ReactToastify.css';
 export const NotesPage=()=>{
     return(
         <>
         <input type="checkbox" id="check"></input>
```

Comment:
as input is not a container tag you can simply write `<input type="checkbox" id="check"/>` (without the closing tag)